mkt/langpacks/models.py

```diff
@@ -81,21 +84,55 @@ def handle_file_operations(self, upload):
         """Handle file operations on an instance by using the FileUpload object
         passed to set filename, size, hash on the LangPack instance, and moving
         the temporary file to its final destination."""
-        # FIXME: store validation result ?
         upload.path = smart_path(nfd_str(upload.path))
         if not self.uuid:
             self.reset_uuid()
         self.filename = self.generate_filename()
```

Comment:
Like with `genereate_hash()`, we could pass what it needs (`uuid`, `version`) as arguments, but I don't see a strong need to do so. Do you ?